src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -36,17 +41,27 @@ public static PreSubmitCallbackResponse<SscsCaseData> processPostponementRequest
                                                                                                              GenerateFile generateFile,
                                                                                                              String templateId) {
 
+        log.debug("Executing processPostponementRequestPdfAndSetPreviewDocument for caseId: {}", sscsCaseData.getCcdCaseId());
```

Comment:
(M) why add this debug line?

Reply:
Just informing on which case we are dealing with. I can remove it if unnecessary.

Reply:
This was done to take into account the (probably impossible) situation where there would have been an NPE.